src/dev-spaces/azext_dev_spaces/azext_metadata.json

```diff
@@ -1,4 +1,3 @@
 {
-    "azext.maxCliCoreVersion": "2.0.66",
-    "azext.minCliCoreVersion": "2.0.63"
+    "azext.minCliCoreVersion": "2.0.67"
```

Comment:
You can keep the min CLI core version as 2.0.63. Because min_profile is removed from your code. So it works for both 2.0.66 and 2.0.67

Reply:
Got it, just pushed an update. Thanks!